### PR TITLE
Add RootCAs as a configurable value to Traefik chart

### PR DESCRIPTION
**Pull request by rafaelmagu**
**What this PR does / why we need it**: It allows a user of the Traefik chart to specify one or more custom certificate authorities to use when connecting to TLS-encrypted backends.

### CODE REVIEW

**Comment by rafaelmagu** [comment]
Something that's a bit unclear to me is the format that `rootCAs` takes when specifying certificates inline. Attempting to add the base64-encoded certificate yields:

```
2018/09/23 21:07:19 Error reading TOML config file /config/traefik.toml : toml: cannot load TOML value of type string into a Go boolean
```

File: stable/traefik/values.yaml [hunk 1]
```diff
@@ -13,6 +13,8 @@ podDisruptionBudget: {}
   # maxUnavailable: 1
   # minAvailable: 2
 
+rootCAs: []
```
Comment (dtomcej) [inline stable/traefik/values.yaml]: This should be commented out by default :)

**Review by dtomcej — CHANGES_REQUESTED** [review]
Can you please bump the chart in addition?

**Comment by rafaelmagu** [comment]
@dtomcej thank you for the feedback. Rebased and signed off with the change and a bump.

**Review by dtomcej — APPROVED** [review]

**Pull request merged**